browser-extensions/firefox/manifest.json

```diff
@@ -70,83 +69,110 @@
+        "*://www.parkrun.co.nl/parkrunner/*/all/",
+        "*://www.parkrun.co.nz/parkrunner/*/all/",
+        "*://www.parkrun.co.za/parkrunner/*/all/",
+        "*://www.parkrun.com.au/parkrunner/*/all/",
+        "*://www.parkrun.com.de/parkrunner/*/all/",
+        "*://www.parkrun.dk/parkrunner/*/all/",
+        "*://www.parkrun.fi/parkrunner/*/all/",
+        "*://www.parkrun.fr/parkrunner/*/all/",
+        "*://www.parkrun.ie/parkrunner/*/all/",
+        "*://www.parkrun.it/parkrunner/*/all/",
+        "*://www.parkrun.jp/parkrunner/*/all/",
+        "*://www.parkrun.lt/parkrunner/*/all/",
+        "*://www.parkrun.my/parkrunner/*/all/",
+        "*://www.parkrun.no/parkrunner/*/all/",
+        "*://www.parkrun.org.uk/parkrunner/*/all/",
+        "*://www.parkrun.pl/parkrunner/*/all/",
+        "*://www.parkrun.ru/parkrunner/*/all/",
+        "*://www.parkrun.se/parkrunner/*/all/",
+        "*://www.parkrun.sg/parkrunner/*/all/",
+        "*://www.parkrun.us/parkrunner/*/all/"
+      ],
+      "js": [
+        "js/lib/third-party/jquery/jquery-3.6.0.js",
+        "js/lib/cache.js",
+        "js/lib/i18n.js",
+        "js/content-scripts/content-script-parkrunner-summary.js"
+      ],
+      "run_at": "document_end"
+    }
   ],
   "web_accessible_resources": [
-      "/images/badges/*.png",
-      "/images/logo/*.png",
-      "/images/maps/markers/*.png",
-      "/images/flags/*.png",
-      "/html/*.html",
-      "/css/*"
+    {
+      "matches": [
+        "https://www.parkrun.ca/*",
+        "https://www.parkrun.co.at/*",
+        "https://www.parkrun.co.nl/*",
+        "https://www.parkrun.co.nz/*",
+        "https://www.parkrun.co.za/*",
+        "https://www.parkrun.com.au/*",
+        "https://www.parkrun.com.de/*",
+        "https://www.parkrun.dk/*",
+        "https://www.parkrun.fi/*",
+        "https://www.parkrun.fr/*",
+        "https://www.parkrun.ie/*",
+        "https://www.parkrun.it/*",
+        "https://www.parkrun.jp/*",
+        "https://www.parkrun.lt/*",
+        "https://www.parkrun.my/*",
+        "https://www.parkrun.no/*",
+        "https://www.parkrun.org.uk/*",
+        "https://www.parkrun.pl/*",
+        "https://www.parkrun.ru/*",
+        "https://www.parkrun.se/*",
+        "https://www.parkrun.sg/*",
+        "https://www.parkrun.us/*"
+      ],
+      "resources": [
+        "/images/badges/*.png",
+        "/images/logo/*.png",
+        "/images/maps/markers/*.png",
+        "/images/flags/*.png",
+        "/html/*.html",
+        "/css/*"
+      ],
+      "use_dynamic_url": true
+    }
   ],
   "permissions": [
     "storage"
   ],
   "options_ui": {
-      "page": "html/options.html",
-      "chrome_style": true
+    "page": "html/options.html"
   },
   "background": {
-    "persistent": true,
-    "scripts": [
-        "js/lib/third-party/jquery/jquery-3.6.0.js",
-        "js/lib/i18n.js",
-        "js/background.js"
-    ]
+    "service_worker": "js/background.js",
```

Comment:
☝️ This is ignored by Firefox but needed by Chrome.
👇 This is rejected by Chrome but needed by Firefox. 🤦